docs/asyncio_working_with_async.md

```diff
@@ -101,6 +101,13 @@ If you are running inside an async context, it might sometimes be necessary to c
 result = await hass.async_add_executor_job(hub.update)
 ```
 
+Or, for a function with parameters:
```

Comment:
Instead of this, update the existing example to add a parameter. Also would be nice to add how that is equivalent to calling `hub.update("some_parameter")`. Might also want to mention that people can use `functools.partial` if they need to pass keyword args.